rllib/algorithms/algorithm_config.py

```diff
@@ -1838,11 +1855,10 @@ def env_runners(
                 be used to collect and retrieve environment-, model-, and sampler data.
                 Override the SampleCollector base class to implement your own
                 collection/buffering/retrieval logic.
-            create_env_on_local_worker: When `num_env_runners` > 0, the driver
-                (local_worker; worker-idx=0) does not need an environment. This is
-                because it doesn't have to sample (done by remote_workers;
-                worker_indices > 0) nor evaluate (done by evaluation workers;
-                see below).
+            create_local_env_runner: If True, create a local EnvRunner instance, besides
```

Comment:
Dumb question: Do we need the local EnvRunner anyways then anymore?

Reply:
Trying to get rid of it for good. But for now leaving this still as optional ...